keyboards/angel17/rules.mk

```
@@ -0,0 +1,44 @@
+# MCU name
+MCU = atmega32u4
+
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   atmega32a    bootloadHID
+BOOTLOADER = atmel-dfu
```

The readme mentions
>Hardware Supported: angel17_alpha, promicro

If so, the bootloader should be updated to match.

```suggestion
BOOTLOADER = caterina
```